blocks/banner/banner.css

```diff
@@ -0,0 +1,46 @@
+main .banner {
+    opacity: 0;
+    max-width: 600px;
+    min-height: 152px;
+    display: flex;
+    padding: 1.5rem 0;
+    border-top: 2px solid #ddd;
+    border-bottom: 2px solid #ddd;
+    align-items: center;
+    margin: 64px auto;
+}
+main .banner.is-loaded {
+    opacity: 1;
+    min-height: 0;
+}
+main .banner img {
+    width: 100px;
+    max-width: unset;
+}
+main .banner > *:nth-child(2) {
+    padding-left: 2rem;   
+}
+main .banner > *:nth-child(2) h1,
+main .banner > *:nth-child(2) h2,
+main .banner > *:nth-child(2) h3,
+main .banner > *:nth-child(2) h4,
+main .banner > *:nth-child(2) h5 {
+    font-size: 14px;
+    font-weight: 700;
+    line-height: 1.5em;
+    letter-spacing: .1em;
+    text-transform: uppercase;
+    margin: .5em 0;
+}
+main .banner > *:nth-child(2) p {
+    margin: .5rem 0;
+    line-height: 1.5rem;
+    font-size: 1rem;
+}
+main .banner > *:nth-child(2) a:any-link {
+    font-weight: 700;
+    text-decoration: none;
+}
```

Comment:
@davidnuescheler I didn't change the color as blue because I believe it should be treated globally.